Source/AviationWeather/Inc/AviationWeather/components.h

```diff
@@ -0,0 +1,180 @@
+/**********************************************************************************
+ *                                                                                *
+ * Copyright (c) 2015 Steven Frost, Orion Lyau. All rights reserved.              *
```

Comment:
Should we change the year since this is a new file?  Also, should we adjust the license in the root directory of the repository?